SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/OwnershipLockingPolicy.cpp

```diff
@@ -115,6 +115,16 @@ bool UOwnershipLockingPolicy::IsLocked(const AActor* Actor) const
 	return false;
 }
 
+int32 UOwnershipLockingPolicy::GetActorLockCount(const AActor* Actor) const
+{
+	if (!ActorToLockingState.Contains(Actor))
+	{
+		return 0;
+	}
+
+	return ActorToLockingState.FindChecked(Actor).LockCount;
```

Comment:
You're hashing this twice. Use find and check for nullptr.

Reply:
I'll hash you twice